static/app/components/alert.tsx

```diff
@@ -73,6 +73,11 @@ const alertStyles = ({
       text-decoration-style: solid;
     }
 
+    pre {
+      background: ${alertColors.backgroundLight};
+      margin: ${space(0.5)} 0 0;
+    }
+
```

Comment:
<img width="848" alt="Screen Shot 2022-04-04 at 11 31 28 AM" src="https://user-images.githubusercontent.com/44172267/161609742-922a3fb8-dd83-42d3-9d9c-d339947da5b7.png">

Reply:
Oh I like this lil detail 🌮